src/Servers/Kestrel/Core/src/CoreStrings.resx

```diff
@@ -548,8 +548,8 @@ For more information on configuring HTTPS see https://go.microsoft.com/fwlink/?l
   <data name="RequestTrailersNotAvailable" xml:space="preserve">
     <value>The request trailers are not available yet. They may not be available until the full request body is read.</value>
   </data>
-  <data name="Http2NoTlsOsx" xml:space="preserve">
-    <value>HTTP/2 over TLS is not supported on macOS due to missing ALPN support.</value>
+  <data name="Http2NoTlsAlpn" xml:space="preserve">
+    <value>HTTP/2 over TLS is not supported on this platform due to missing ALPN support.</value>
```

Comment:
I dunno if showing `RuntimeInformation.OSDescription` would make sense here, but maybe?

Reply:
Maybe the error shouldn't mention "this platform" at all. There could be other hypothetical reasons why ALPN isn't supported.